src/Parser/AnonymousClassVisitor.php

```diff
@@ -27,10 +27,20 @@ public function enterNode(Node $node): ?Node
 			return null;
 		}
 
-		$node->setAttribute(self::ATTRIBUTE_ANONYMOUS_CLASS, true);
```

Comment:
In theory this isn't covered by the backward compatibility promise, right?
Even so, the removal of this would cause some breakages (see e.g. https://github.com/mglaman/phpstan-drupal/pull/195)

Reply:
Let's keep the attribute but don't use it ourselves anywhere anymore.